examples/udp_socket-rs/ta/src/main.rs

```diff
@@ -74,13 +122,33 @@ fn udp_socket() {
                 response.extend_from_slice(&chunk[..n]);
                 break;
             }
-            Err(_) => {
-                trace_println!("Error");
-                panic!();
+            Err(err) => {
+                trace_println!("failed to read due to {:?}", err);
+                return Err(ErrorKind::Generic.into());
             }
         }
     }
     trace_println!("{}", String::from_utf8_lossy(&response));
+    Ok(())
 }
 
 include!(concat!(env!("OUT_DIR"), "/user_ta_header.rs"));
+
+/// Workaround for those rustc bugs:
+/// * https://github.com/rust-lang/rust/issues/47493
```

Comment:
Have you tries this workaround in the issue? https://github.com/rust-lang/rust/issues/47493#issuecomment-606385776